cheyenne_gnu_fms_mpp_util_mpi_inc.patch

```diff
@@ -1,12 +0,0 @@
---- FMS/mpp/include/mpp_util_mpi.inc	2021-02-08 08:24:21.000000000 -0700
-+++ FMS/mpp/include/mpp_util_mpi.inc	2021-02-08 08:24:15.000000000 -0700
```

Comment:
So this patch is no longer needed or is moved to fms library?

Reply:
We don't know until we try. If it is still needed, it will have to go to hpc-stack so that we have it under our own control.

Reply:
It will be difficult to apply inside of hpc-stack build_fms.sh. 
There are no machine specific conditionals in any of these build scripts.